integration_tests/testsuite/test_util.py

```diff
@@ -23,47 +23,23 @@
 import requests
 from retrying import retry
 import string
-import subprocess
-import sys
+from subprocess import Popen, PIPE, check_output, CalledProcessError
 import google.auth
 
-requests.packages.urllib3.disable_warnings()
-
-LOGNAME_LENGTH = 16
-
-LOGGING_PREFIX = 'GCP_INTEGRATION_TEST_'
-
-DEFAULT_TIMEOUT = 30  # seconds
-
-ROOT_ENDPOINT = '/'
-ROOT_EXPECTED_OUTPUT = 'Hello World!'
-
-STANDARD_LOGGING_ENDPOINT = '/logging_standard'
-CUSTOM_LOGGING_ENDPOINT = '/logging_custom'
-MONITORING_ENDPOINT = '/monitoring'
-EXCEPTION_ENDPOINT = '/exception'
-CUSTOM_ENDPOINT = '/custom'
+import constants
 
-METRIC_PREFIX = 'custom.googleapis.com/{0}'
-METRIC_TIMEOUT = 60  # seconds
-
-# subset of levels found at
-# https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#logseverity
-SEVERITIES = [
-    'WARNING',
-    'ERROR',
-    'CRITICAL'
-]
+requests.packages.urllib3.disable_warnings()
 
 
 def _generate_name():
     name = ''.join(random.choice(string.ascii_uppercase +
-                   string.ascii_lowercase) for i in range(LOGNAME_LENGTH))
+                   string.ascii_lowercase)
+                   for i in range(constants.LOGNAME_LENGTH))
     return name
 
 
-def _generate_hex_token():
-    return binascii.b2a_hex(os.urandom(16))
+def _generate_hex_token(num_bytes):
```

Comment:
This is fine, but FYI there is a `uuid` module if you want to use that.

Reply:
ah yep. I'll leave this as is since I know it works, but good to know for the future.